src/agent/CommandLockAll.ts

```diff
@@ -46,17 +45,27 @@ class CommandLockAll extends CommandPolykey {
       let pkClient: PolykeyClient;
       this.exitHandlers.handlers.push(async () => {
         if (pkClient != null) await pkClient.stop();
-        if (webSocketClient != null) await webSocketClient.destroy(true);
+        if (webSocketClient != null) {
+          await webSocketClient.destroy({ force: true });
+        }
       });
       try {
         webSocketClient = await WebSocketClient.createWebSocketClient({
-          expectedNodeIds: [clientOptions.nodeId],
+          config: {
+            verifyPeer: true,
+            verifyCallback: async (certs) => {
+              await clientUtils.verifyServerCertificateChain(
+                [clientOptions.nodeId],
+                certs,
+              );
+            },
+          },
           host: clientOptions.clientHost,
           port: clientOptions.clientPort,
           logger: this.logger.getChild(WebSocketClient.name),
         });
```

Comment:
Shouldn't `PolykeyClient` encapsulate the WS client setup. I don't like having every command setup its own websocket transport. That seems very incorrect.

Reply:
I think we wanted `PolykeyClient` to be agnostic to transport like the RPC?

Reply:
No of course not!

Reply:
How can PK client be agnostic, and yet PK agent be not agnostic. Doesn't make sense.

Reply:
@amydevs can you help on this problem?

Reply:
Ok once `WebSocketClient` is moved into the `PolykeyClient`. That should mean `@matrixai/ws` should be removed from the `package.json`.